### DISCUSSION TITLE
Period of record and water year

### DISCUSSION THREAD

**Discussion by slevin75** [USGS-R/regional-hydrologic-forcings-ml#6] [General]
Some questions regarding period of record and water year.  I have been assuming we are doing this on a water-year basis not calendar year, so I have added that parameter to the top of the targets.R file, but Eflowstats can do either one.

Also - I have put 9/30/2020 as the endDate just because there may be a lot of provisional or missing data after that and I arbitrarily put 10/1/1900 as a start date.  Is there a fixed or a minimum period of record we want to consider? We will probably need to screen for some minimum number of years of complete data because there will be at least a few sites that have none or only a few years of complete data and I think we'll want at least 10 or 15 years of data.  or more?

**Comment by jds485** [comment]
I am in favor of using water year.

I also agree with ending 9/30/2020 due to provisional data in 2021. 
For now, let's keep the start date as early as possible. We can do a moving window analysis for each gauge to see how metrics are changing over time. If there are substantial changes for some sites, I think that can help inform if we should limit our analysis to a more recent time period. Otherwise, a longer record will help to obtain more robust statistics (metrics). I'll start an issue to set up the moving window analysis for metrics.

I think the GAUGES database already screens sites for minimum record length, but I'm not sure what that length is. We can revisit when the GAUGES III database is available.
